packages/social-trader/src/components/center/center.tsx

```diff
@@ -11,6 +11,7 @@ interface Props extends React.HTMLAttributes<HTMLDivElement> {
   small?: boolean;
   middle?: boolean;
   large?: boolean;
+  justifyCenter?: boolean;
```

Comment:
Component closed for modification. Extend Center and change justify-content in extended component.

Reply:
Done